CONTRIBUTING.adoc

```diff
@@ -93,7 +93,14 @@ NOTE: Replace "linux/amd64" with the appropriate value for your platform/archite
 
         $ sudo chmod +r openshift.local.config/master/openshift-registry.kubeconfig
         $ sudo chmod +r openshift.local.config/master/admin.kubeconfig
-        $ oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
+        $ sudo ./oadm registry --create --credentials=openshift.local.config/master/openshift-registry.kubeconfig --config=openshift.local.config/master/admin.kubeconfig
```

Comment:
hmm I recall the ansible set up requiring Sudo because of master certs that were created via sudo openshift start?  Maybe I'm wrong though. In my setup it was required .